Library/ViewModels/RewardsCollectionViewModel.swift

```diff
@@ -273,7 +271,9 @@ private func titleForContext(_ context: RewardsCollectionViewContext, project: P
 }
 
 private func shouldNavigateToReward(project: Project, reward: Reward, refTag _: RefTag?) -> Bool {
-  project.state == .live && (!userIsBacking(reward: reward, inProject: project) || reward.hasAddOns)
+  guard !currentUserIsCreator(of: project) else { return false }
```

Comment:
Do we have a test for this guard? Might be worth adding another test to check for this?

Reply:
Yeah that's the one below? `testGoToPledge_DoesNotEmitIfCreatorOfProject` or was there another case you wanted to test for?

Reply:
Ah, I missed the test for the creator, thank you 👍